tests/test_basic.py

```diff
@@ -3,13 +3,19 @@
 
 import os
 import sys
-from unittest import TestCase
+from unittest import TestCase, skipIf
 
 from nose.plugins import PluginTester
 from nose_randomly import RandomlyPlugin
 
 fixtures = os.path.join(os.path.dirname(__file__), 'fixtures')
 
+try:
```

Comment:
I didn't add it there. The test should only be executed if numpy is in the current environment